src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/CertificatePal.PrivateKey.cs

```diff
@@ -417,9 +415,10 @@ private unsafe ICertificatePal CopyWithPersistedCngKey(CngKey cngKey)
             }
             catch (CryptographicException e)
```

Comment:
Weird, maybe I didn't recompile after removing the Debug.WriteLine I had.  Oops.